docs/build-matrix.html

```diff
@@ -41,119 +41,119 @@ <h2>MXE Build Matrix</h2>
  <th class="row" title=""> a52dec   </th>
  <td>0.7.4 </td>
       <td class="supported">&#x2713;</td>
-      <td class="supported">&#x2713;</td>
```

Comment:
Don't regenerate this file - the MXEBot does that when merged.

Reply:
Ok my bad

Reply:
> the MXEBot does that when merged

Maybe not. From https://travis-ci.org:

>Since June 15th, 2021, the building on [travis-ci.org](http://www.travis-ci.org/) is ceased. Please use [travis-ci.com](http://www.travis-ci.com/) from now on.

Reply:
Yeah maybe you should consider github actions. This is not demanding at all. Just to run that make docs command. You can directly push that to a feature branch, triggered only with a PR.